bin/sdk_generator.sh

```diff
@@ -25,7 +25,7 @@
 
 set -e
 
-GLEAN_PARSER_VERSION=6.1
+GLEAN_PARSER_VERSION=7.0
```

Comment:
Did you mean to update this or is this an automatic update? 🤔

Reply:
I meant to update this. I bumped the version of a-s and v97.0.0 bumps the glean version.